Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionTester.swift

```diff
@@ -222,29 +241,30 @@ final class NetworkProtectionConnectionTester {
         let localParameters = NWParameters.tcp
         localParameters.prohibitedInterfaces = [tunnelInterface]
 
-        Task {
```

Comment:
The diff below looks a bit ugly, but for the most part is just a tab-level change.